### ISSUE TITLE
(Feature request) aquadynamic upgrade

### ISSUE THREAD

**Issue by Raikto** [WayofTime/BloodMagic#742]
It could be cool if our living armor could be useful to explore ocean monuments for instance. With an upgrade we could swim faster or get a better vision underwater. That s really neat for whose build underwater cities.

**Comment by Arcaratus** [comment]
Herm...swimming sigil....hermm....

By the Way, are aquatic upgrades planned for Living Armour?

**Comment by WayofTime** [comment]
No, they weren't planned - I will have to see how I can implement it, since vanilla hard-codes stuff like water movement resistance and the vision blur.

**Comment by Arcaratus** [comment]
If you want, [this](https://github.com/Arcaratus/BloodArsenal/blob/master/src/main/java/com/arc/bloodarsenal/common/BloodArsenalEventHooks.java#L145) could be helpful in the water movement (Sigil of Swimming code). The water blur however, I checked when I was writing the swimming stuff, and I have not figured out a way to do that without hacky things. It's hardcoded within the player's render

**Comment by mywither** [comment]
Will the Sigil of Suppression not be helpful enough removing the water around you so you can just walk?

**Comment by keraldi** [comment]
1.13+? (aquatic update)

**Comment by MentalMouse** [comment]
In practice, I find the Elemental Sigil (for water breathing) and any consistent form of Night Vision are enough to let me wander underwater at will.  The Sigil of Haste may help a bit (didn't have it for my latest underwater expedition), but the Sigil of Air works fine underwater.  Having water breathing and underwater vision on the Living Armor would be a convenience, but a specialized one.

The Sigil of Suppression actually cuts into my mobility, because I'm limited to walking (or SoA) and bound by gravity.  That said, it cuts into the Guardians' mobility even more.   ;-)
